code/ModelAdmin.php

```diff
@@ -183,6 +185,55 @@ public function Link($action = null)
         return parent::Link($action);
     }
 
+    /**
+     * Get the link for the tab of a DataObject subclass managed by this ModelAdmin.
+     *
+     * Note that if the modelClass is managed on multiple tabs and you want a link for a
+     * specific tab you should use {@link getLinkForModelTab} instead.
+     *
+     * @throws InvalidArgumentException if $modelClass is not managed by this ModelAdmin.
+     */
+    public function getLinkForModelClass(string $modelClass): string
+    {
+        if (!$this->isManagedModel($modelClass)) {
+            throw new InvalidArgumentException("$modelClass isn't managed by this ModelAdmin.");
+        }
+        return $this->getLinkForModelTab($this->getModelTabForModelClass($modelClass));
+    }
+
+    /**
+     * Get the link for a specific tab on this ModelAdmin.
+     *
+     * If you do not have multiple tabs for a given class you should use {@link getLinkForModelClass}
+     * instead.
+     *
```

Comment:
The whole "modelTab" concept it pretty confusing, I think it's worth clarifying in the method comment that modelTab can either be:
- the url-slug of the modelTab - the reason why this method exists
- the modelClass of the modelTab - and yes this is functionally the same as getLinkForModelClass()

Reply:
Added messaging to the phpdoc